kratos/tests/test_processes.py

```diff
@@ -2521,6 +2521,84 @@ def test_split_internal_interfaces_process_3D(self):
         self.assertEqual(model_part.GetCondition(10).GetGeometry()[0].Id, 33)
         self.assertEqual(model_part.GetCondition(10).GetGeometry()[1].Id, 2)
         self.assertEqual(model_part.GetCondition(10).GetGeometry()[2].Id, 31)
+    
+    def test_assign_mpcs_to_neighbours_process(self):
+        # Set up the model
+        current_model = KratosMultiphysics.Model()
+        model_part = current_model.CreateModelPart("Main")
+        model_part.AddNodalSolutionStepVariable(KratosMultiphysics.DISPLACEMENT)
+        model_part_io = KratosMultiphysics.ModelPartIO(GetFilePath("auxiliar_files_for_python_unittest/mdpa_files/test_assign_mpcs_to_neighbours_process"))
+        model_part_io.ReadModelPart(model_part)
+        KratosMultiphysics.VariableUtils().AddDof(KratosMultiphysics.DISPLACEMENT_X, model_part)
+
+        # Set up process parameters
+        settings = KratosMultiphysics.Parameters(
+            """
+            {
+                "process_list" : [
+                    {
+                        "python_module": "assign_mpcs_to_neighbours_process",
+                        "kratos_module": "KratosMultiphysics",
+                        "process_name": "AssignMPCsToNeighboursProcess",
+                        "Parameters": {
+                            "computing_model_part_name": "Main",
+                            "master_model_part_name": "Main.GENERIC_Master",
+                            "slave_model_part_name": "Main.GENERIC_Slave",
+                            "search_radius": 0.01,
+                            "minimum_number_of_neighbouring_nodes": 3,
+                            "assign_mpcs_every_time_step":  true,
+                            "variable_names": ["DISPLACEMENT_X"]
+                        }
+                    }
+                    ]
+                }
+            """
+        )
+
+        # Create and execute processes
+        list_of_processes = process_factory.KratosProcessFactory(current_model).ConstructListOfProcesses(settings["process_list"])
+        for process in list_of_processes:
+            process.ExecuteInitialize()
+            process.ExecuteInitializeSolutionStep()

```

Comment:
You have a unique process. I don't see the point on doing it with a list. I'd directly call the constructor.